src/matlab2tikz.m

```diff
@@ -2848,7 +2850,7 @@ function assertRegularAxes(handle)
         annotPanes   = findall(m2t.currentHandles.gcf,'Tag','scribeOverlay');
         children = allchild(annotPanes);
         if iscell(children)
-            children = [children{:}]
```

Comment:
Good find. I don't know whether that path is covered, maybe we should place a `TODO: is this dead code?` here to doublecheck that?